qiskit/circuit/quantumcircuit.py

```diff
@@ -3521,6 +3521,17 @@ def count_ops(self) -> "OrderedDict[Instruction, int]":
             count_ops[instruction.operation.name] = count_ops.get(instruction.operation.name, 0) + 1
         return OrderedDict(sorted(count_ops.items(), key=lambda kv: kv[1], reverse=True))
 
+    # The stringified return type is because OrderedDict can't be subscripted before Python 3.9, and
+    # typing.OrderedDict wasn't added until 3.7.2.  It can be turned into a proper type once 3.6
+    # support is dropped.
+    def count_ops_with_qubits(self) -> list:
```

Comment:
The next qiskit release will only support python >= 3.9 so this isn't relevent. Also this isn't currently returning an ordered dict. Although I would argue we should not be using an ordered dict at all. The other methods do because they have been around since Qiskit supported much older versions of python (before dicts were sorted by default) and have retained it for backwards compatibility. But for new functionality we should just return a dict or a list.

```suggestion
```